.github/actions/tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+# Add python libraries
+sudo apt update
+sudo apt install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libsqlite3-dev libreadline-dev libffi-dev wget libbz2-dev
+sudo add-apt-repository ppa:deadsnakes/ppa -y
+sudo apt install python3.7
+sudo apt-get install python3-lxml xmlstarlet
+pip install lxml
+
+
+git clone -b ${DATASET_BRANCH} https://github.com/pkp/jatsTemplate plugins/generic/jatsTemplate
```

Comment:
Shouldn't we always be checking out datasets `main` branch?

Reply:
@asmecher 
The name here was misleading, here I target the corresponding similar branch in the jatsTemplate plugin for the oaiJats plugin.  I changed the variable to ${APP_BRANCH}